src/ContactComponent.php

```diff
@@ -162,19 +162,15 @@ function wf_crm_contact_search($node, $component, $params, $contacts, $str = NUL
         }
       }
       // If we get exactly $limit results, there are probably more - warn that the list is truncated
-      // TODO Error reporting is still Drupal7-ish
       if (wf_crm_aval($result, 'count') >= $limit) {
-        watchdog(
-          'webform_civicrm',
-          'Maximum contacts exceeded, list truncated on the webform "@title". The webform_civicrm "@field" field cannot display more than !limit contacts because it is a select list. Recommend switching to autocomplete widget in component settings.',
-          array('!limit' => $limit, '@field' => $component['name'], '@title' => $node->title),
-          WATCHDOG_WARNING,
-          l(t('Edit component'), "node/{$node->nid}/webform/components/{$component['cid']}")
+        \Drupal::logger('webform_civicrm')->warning(
+          'Maximum contacts exceeded, list truncated on the webform "@title". The webform_civicrm "@field" field cannot display more than @limit contacts because it is a select list. Recommend switching to autocomplete widget in component settings.',
+          array('@limit' => $limit, '@field' => $component['#title'], '@title' => $node->label()),
         );
-        if (wf_crm_admin_access($node) && node_is_page($node)) {
```

Comment:
Verified admin_access is this

```
/**
 * Access callback to determine if user can see the CiviCRM tab of a webform.
 *
 * @param object $node
 * @return bool
 */
function wf_crm_admin_access($node) {
  return (node_access('update', $node) && user_access('access CiviCRM'));
}
```

I thought I left functions like this in `webform_civicrm.module` during the report so we wouldn't have breaks like this :(. Sorry!